legal-api/src/legal_api/resources/v2/business/colin_sync.py

```diff
@@ -31,68 +31,60 @@
 
 
 @bp.route('/internal/filings', methods=['GET'])
-@bp.route('/internal/filings/<string:status>', methods=['GET'])
```

Comment:
In v1 endpoint future effective job was using this. moving this to internal endpoint, so that we can depreciate  `colin_sync.py` once we completely migrate to lear